### ISSUE TITLE
Add new rounding functions 

### ISSUE THREAD

**Issue by saig0** [camunda/feel-scala#277]
**Is your feature request related to a problem? Please describe.**
With DMN 1.4, three new rounding methods will be introduced for rounding up, down, and half up. 

**Describe the solution you'd like**
![image](https://user-images.githubusercontent.com/4305769/120421767-2ed9b700-c367-11eb-852e-e698260711a5.png)

![image](https://user-images.githubusercontent.com/4305769/120421810-49ac2b80-c367-11eb-823f-58a4afb5c020.png)

![image](https://user-images.githubusercontent.com/4305769/120421825-516bd000-c367-11eb-9db2-9e26bb6de9b0.png)

**Related issues**

* Camunda BPM: 
* Zeebe broker: 
* OMG: https://issues.omg.org/browse/DMN14-126

**Comment by kaaquist** [comment]
Hi is this here up for grabs?

**Comment by kaaquist** [comment]
Perfect, thanks.
I had a look at the solution description above, and is it right that the `round up` feature will be doing `ceiling` for positive numbers but `floor` for negative numbers? So `round up(5.5, 0) = 6` and `round up(-5.5, 0) = -6` ? . 
Same goes for `round down` where we "just" switch the `ceiling` and `floor` on the negative and positive numbers?  
So `round down(5.5, 0) = 5` and `round down(-5.5, 0) = -5` ?

**Comment by saig0** [comment]
> is it right that the round up feature will be doing ceiling for positive numbers but floor for negative numbers?

Yes, exactly. In the code, we can use Java's building `RoundingMode.UP` and `RoundingMode.DOWN`.
